workbench/src/main/setupAddRemovePlugin.js

```diff
@@ -11,59 +11,98 @@ import { settingsStore } from './settingsStore';
 
 const logger = getLogger(__filename.split('/').slice(-1)[0]);
 
+function spawnWithLogging(cmd, args, options) {
```

Comment:
Should this function have a docstring above it?

Reply:
Added a docstring